nipype/interfaces/freesurfer/petsurfer.py

```diff
@@ -0,0 +1,584 @@
+        desc="Subcortical GM voxel-wise values corrected using the extended Muller-Gartner method",
+    )
+    mgx_gm = File(
+        desc="All GM voxel-wise values corrected using the extended Muller-Gartner method",
+    )
+    rbv = File(desc="All GM voxel-wise values corrected using the RBV method")
+    opt_params = File(
+        desc="Optimal parameter estimates for the FWHM using adaptive GTM"
+    )
+    yhat0 = File(desc="4D PET file of signal estimate (yhat) after PVC (unsmoothed)")
+    yhat = File(
+        desc="4D PET file of signal estimate (yhat) after PVC (smoothed with PSF)",
+    )
+    yhat_full_fov = File(
+        desc="4D PET file with full FOV of signal estimate (yhat) after PVC (smoothed with PSF)",
+    )
+    yhat_with_noise = File(
+        desc="4D PET file with full FOV of signal estimate (yhat) with noise after PVC (smoothed with PSF)",
+    )
+
+
+class GTMPVC(FSCommand):
+    """create an anatomical segmentation for the geometric transfer matrix (GTM).
+
+    Examples
+    --------
+    >>> gtmpvc = GTMPVC()
+    >>> gtmpvc.inputs.in_file = 'sub-01_ses-baseline_pet.nii.gz'
+    >>> gtmpvc.inputs.segmentation = 'gtmseg.mgz'
+    >>> gtmpvc.inputs.reg_file = 'sub-01_ses-baseline_pet_mean_reg.lta'
+    >>> gtmpvc.inputs.pvc_dir = 'pvc'
+    >>> gtmpvc.inputs.psf = 4
+    >>> gtmpvc.inputs.default_seg_merge = True
+    >>> gtmpvc.inputs.auto_mask = (1, 0.1)
+    >>> gtmpvc.inputs.km_ref = ['8 47']
+    >>> gtmpvc.inputs.km_hb = ['11 12 50 51']
+    >>> gtmpvc.inputs.no_rescale = True
+    >>> gtmpvc.inputs.save_input = True
+    >>> gtmpvc.cmdline  # doctest: +NORMALIZE_WHITESPACE
+    'mri_gtmpvc --auto-mask 1.000000 0.100000 --default-seg-merge \
+    --i sub-01_ses-baseline_pet.nii.gz --km-hb 11 12 50 51 --km-ref 8 47 --no-rescale \
+    --psf 4.000000 --o pvc --reg sub-01_ses-baseline_pet_mean_reg.lta --save-input \
+    --seg gtmseg.mgz'
+    """
+
+    _cmd = "mri_gtmpvc"
+    input_spec = GTMPVCInputSpec
+    output_spec = GTMPVCOutputSpec
+
+    def _list_outputs(self):
+        outputs = self.output_spec().get()
+        # Get the top-level output directory
+        if not isdefined(self.inputs.pvc_dir):
+            pvcdir = os.getcwd()
+        else:
+            pvcdir = os.path.abspath(self.inputs.pvc_dir)
+        outputs["pvc_dir"] = pvcdir
+
+        # Assign the output files that always get created
+        outputs["ref_file"] = os.path.join(pvcdir, "km.ref.tac.dat")
+        outputs["hb_nifti"] = os.path.join(pvcdir, "km.hb.tac.nii.gz")
+        outputs["hb_dat"] = os.path.join(pvcdir, "km.hb.tac.dat")
+        outputs["nopvc_file"] = os.path.join(pvcdir, "nopvc.nii.gz")
+        outputs["gtm_file"] = os.path.join(pvcdir, "gtm.nii.gz")
+        outputs["gtm_stats"] = os.path.join(pvcdir, "gtm.stats.dat")
+        outputs["reg_pet2anat"] = os.path.join(pvcdir, "aux/bbpet2anat.lta")
+
+        # Assign the conditional outputs
+        if isdefined(self.inputs.save_input) and self.inputs.save_input:
```

Comment:
I believe this can be simplified to just
```suggestion
        if self.inputs.save_input:
```

but worth a check before accepting

Reply:
LGTM after testing.